orchestra/models/communication/mixins.py

```diff
@@ -53,6 +53,7 @@ class StaffingRequestInquiryMixin(object):
 
     def __str__(self):
         return '{} - {}'.format(
+            self.id,
```

Comment:
do you have to change the format string?

Reply:
Nope. It was for debugging. I will remove this.